xml/System.Buffers/IBufferWriter`1.xml

```diff
@@ -67,10 +67,12 @@
         <Parameter Name="sizeHint" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="sizeHint">The size of the <see cref="T:System.Memory`1" /> requested. If 0, requests currently available memory. </param>
-        <summary>Requests a <see cref="T:System.Memory`1" /> that is at least <paramref name="sizeHint" /> in size if possible.</summary>
-        <returns>A <see cref="T:System.Memory`1" /> of size <paramref name="sizeHint" />, if it's available; otherwise, a <see cref="T:System.Memory`1" /> with the maximum available memory. If <paramref name="sizeHint" /> is 0, returns currently available memory.</returns>
-        <remarks>To be added.</remarks>
+        <param name="sizeHint">The minimum length of the returned <see cref="T:System.Memory`1" />. If 0, some, non-empty buffer is returned. </param>
+        <summary>Returns a <see cref="T:System.Memory`1" /> to write to that is at least the requested size (specified by <paramref name="sizeHint" />).</summary>
+        <returns>A <see cref="T:System.Memory`1" /> of at least the size <paramref name="sizeHint" />. If <paramref name="sizeHint" /> is 0, returns a non-empty buffer.</returns>
+        <remarks>There is no guarantee that successive calls will return the same buffer or the same-sized buffer.</remarks>
+        <remarks>This must never return an empty <see cref="T:System.Memory`1" /> but it can throw <see cref="System.OutOfMemoryException"/> if the requested buffer size is not available.</remarks>
```

Comment:
I don't think implementations should throw `OutOfMemoryException` neither I think we should prescribe exception type here at all.

Reply:
So remove the whole exception clause all together OR at least mention that it can throw?

**Option 1:**
```text
<remarks>This must never return an empty <see cref="T:System.Memory`1" />.</remarks>
```

**Option 2:**
```
<remarks>This must never return an empty <see cref="T:System.Memory`1" /> but it can throw if the requested buffer size is not available.</remarks>
```